components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -219,9 +219,15 @@ be lost if SCREAM_HACK_XML is not enabled.
       <nudging_filename type="array(string)"/>
       <nudging_fields type="array(string)"/>
       <nudging_timescale type="integer">0</nudging_timescale>
-      <source_pressure_type type="string">TIME_DEPENDENT</source_pressure_type>
+      <source_pressure_type type="string">TIME_DEPENDENT_3D_PROFILE</source_pressure_type>
     </nudging>
 
+    <!-- ML correction -->
+    <mlcorrection inherit="atm_proc_base">
+      <ML_model_path type="string">NONE</ML_model_path>
+      <ML_output_fields type="array(string)">NONE</ML_output_fields>
```

Comment:
For arrays, you don't need to use a special value for invalid. Just leave an empty param:
```
<ML_output_fields type="array(string)"/>
```

Reply:
You'll have to change the place where this param is pinged in the cpp code as well. Any code of the form
```
if (vec.size()==1 && vec[0]=="NONE") {
  return;
}
```
should be replaced with the more self-explanatory (and less bug prone)
```
if (vec.size()==0) {
  return;
}
```

Reply:
@elynnwu , my changes flagged as changes to the ML correction namelist section.  When we switch to turning ML correction back on for all machines could you also incorporate these suggested changes?

Reply:
Will do.